cli/src/config.rs

```diff
@@ -126,6 +126,10 @@ pub fn read_configuration(path: &Path) -> Result<Config> {
         config.auth_info.offline_access = Some(RefreshToken::new(key));
```

Comment:
This is the line where it is getting set to `Some("")` when an empty string environment variable is used.

If we prevent that assignment (by adding an `if !key.is_empty()` clause around it), then we will use the token from the config, which will probably fix more of the issues that are occurring in the phylum-ci project.

Reply:
I've kept the current change to `None` code for the unlikely case that the configuration file contains `offline_access: ""` due to manual editing. Now if the environment variable is set to empty there should be a warning and the environment value will be ignored.